modules/dnn/src/layers/pooling_layer.cpp

```diff
@@ -230,7 +230,7 @@ class PoolingLayerImpl : public PoolingLayer
         Size kernel, stride, pad;
         int nstripes;
         bool computeMaxIdx;
-        std::vector<int> ofsbuf;
+        AutoBuffer<int> ofsbuf;
```

Comment:
let's keep `std::vector<>`

Reply:
If we need some alignment requirement, we could use `AutoBuffer<int, 0>` to use OpenCV `fastAlloc()`.

Reply:
I think it is OK.